packages/design-system/src/components/N8nFormInputs/FormInputs.vue

```diff
@@ -13,7 +13,6 @@ export type FormInputsProps = {
 	columnView?: boolean;
 	verticalSpacing?: '' | 'xs' | 's' | 'm' | 'l' | 'xl';
 	teleported?: boolean;
-	tagSize?: 'small' | 'medium';
```

Comment:
is this intentiional?

Reply:
Yes, it's just a wrapper component and actually that caused sizing not working properly.
Sizes should be defined in the inputs array on which this component iterates and renders `FormInput.vue` components